channels/kr.m3u

```diff
@@ -165,3 +165,5 @@ http://54.180.143.6:9981/stream/channelid/750850617
 http://54.180.143.6:9981/stream/channelid/1805607194
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/r1WTYTa.png" group-title="",현대홈쇼핑+샵 / + Shop
 http://54.180.143.6:9981/stream/channelid/965307250
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
+http://211.49.191.148:9981/stream/channelid/1158483964
```

Comment:
![afbeelding](https://user-images.githubusercontent.com/25525836/66757918-7adb9d80-ee9d-11e9-9478-75c7880f8969.png)
Link does not work

Reply:
<img width="892" alt="Screen Shot 2019-10-15 at 1 43 28 AM" src="https://user-images.githubusercontent.com/1114030/66785105-4b548180-eeed-11e9-9385-1623a740097e.png">

Reply:
stream works for me. with lags but works (tested with VLC and streamtest)